NWL/NWL.py

```diff
@@ -451,6 +486,41 @@ def NWL_plot(
     # Define number of source particles
     num_parts = len(coords)
 
-    NWL_mat = count_mat*n_energy*eV2J*SS*J2MJ/num_parts
+    neutron_crossing_mat = count_mat*n_energy*eV2J*SS*J2MJ/num_parts
+
+    # construct array of bin boundaries
+    bin_arr = np.zeros((num_phi+1, num_theta+1, 3))
+
+    for phi_bin, phi in enumerate(phi_bins):
+        for theta_bin, theta in enumerate(theta_bins):
+            
+            if theta < theta_min:
+                theta = theta_min
+            elif theta > theta_max:
+                theta = theta_max
+
+            if phi < phi_min:
+                phi = phi_min
+            elif phi > phi_max:
+                phi = phi_max
```

Comment:
First, why would this happen?  Doesn't the above code prevent this? If not, can't we prevent it?

Second, why not:
```
theta = min(max(theta_min, theta), theta_max)
phi = min(max(phi_min, phi), phi_max)
```

or even better? do this with numpy arrays outside of a loop:
```
theta_bins = min(max(theta_bins, theta_min), theta_max)
phi_bins = min(max(phi_bins, phi_min), phi_max)
```

Reply:
You are correct, the 2d histogram should prevent this from occurring, thank you